.github/workflows/tests.yml

```diff
@@ -29,7 +29,10 @@ jobs:
           - { python: "3.11", os: "windows-2022", session: "tests" }
           - { python: "3.11", os: "ubuntu-22.04", session: "tests" }
           - { python: "3.11", os: "macos-12", session: "tests" }
-          - { python: "3.10", os: "ubuntu-22.04", session: "docs-build" }
+          - { python: "3.12", os: "windows-2022", session: "tests" }
```

Comment:
I think it would be good to update the precommit and mypy checks to run with 3.11 now?

Reply:
Just changed!